tests/test_token_gen.py

```diff
@@ -522,20 +579,27 @@ class TestVerifySessionCookie:
         'IDToken': TEST_ID_TOKEN,
     }
 
+    cookies_not_invalid_in_emulator = [
```

Comment:
Let's call this `invalid_cookies_accepted_in_emulator` to avoid the double negative.

Reply:
That's what I originally had, but it breaks the linter as the name's too long:
```
tests/test_token_gen.py:428:4: C0103: Class attribute name "invalid_tokens_accepted_in_emulator" doesn't conform to '([A-Za-z_][A-Za-z0-9_]{2,30}|(__.*__))$' pattern (invalid-name)
tests/test_token_gen.py:583:4: C0103: Class attribute name "invalid_cookies_accepted_in_emulator" doesn't conform to '([A-Za-z_][A-Za-z0-9_]{2,30}|(__.*__))$' pattern (invalid-name)
```

(Same problem with the slightly shorter variant `invalid_tokens_valid_in_emulator`)

Reply:
How about just `cookies_accepted_in_emulator` and `tokens_accepted_in_emulator`?

Reply:
Still feels not quite right to me, but I don't want to bikeshed on this, so `*_accepted_in_emulator` it is.

Reply:
I understand the concern. It's a compromise for sure.